.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -13,7 +13,7 @@ assignees: ''
 **Expected behavior**
```

Comment:
capital maybe?

Reply:
Changed it to lower-case to make it fit with the other titles :laughing: